src/amo/components/UserProfile/index.js

```diff
@@ -295,8 +422,8 @@ export class UserProfileBase extends React.Component<Props> {
 }
 
 export function mapStateToProps(
-  state: {| users: UsersStateType |},
-  ownProps: Props,
+  state: {| reviews: ReviewState, users: UsersStateType |},
+  ownProps: InternalProps,
```

Comment:
This should be `Props` because `mapStateToProps` only receives the initial component. Did you get an error? I only see a reference to `ownProps.params` which is in `Props`.

Reply:
ah no, no error. good catch